examples/MicroserviceExample/Utils/Messaging/MessageReceiver.cs

```diff
@@ -90,6 +95,7 @@ public void ReceiveMessage(BasicDeliverEventArgs ea)
 
         private IEnumerable<string> ExtractTraceContextFromBasicProperties(IBasicProperties props, string key)
         {
+            Console.WriteLine($"Extracting '{key}' from carrier");
```

Comment:
Adding this line just to see when context extraction occurs. This line is hit on the second message received.